camel/personas/persona_generator.py

```diff
@@ -0,0 +1,230 @@
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+import os
+import re
+import uuid
+from typing import Dict, List, Optional, Union
+
+from camel.agents import ChatAgent
+from camel.configs import ChatGPTConfig
+from camel.messages import BaseMessage
+from camel.models import BaseModelBackend, ModelFactory
+from camel.personas import Persona
+from camel.prompts import TextPrompt
+from camel.types import ModelPlatformType, ModelType, RoleType
+
+
+class PersonaGenerator(ChatAgent):
```

Comment:
I feel we don't need to set ChatAgent as the parent class for PersonaGenerator. The purpose and functionality of these two classes are quite different